archetypes/helidon/src/main/archetype/se/custom/files/src/test/java/__pkg__/FtServiceTest.java.mustache

```diff
@@ -172,9 +173,9 @@ class FtServiceTest {
         assertThat(response, is("Slept for 50 ms"));
 
         WebClientResponse clientResponse = client.get()
-                .path("/timeout/105")
+                .path("/timeout/10000")
```

Comment:
This is I think not correct - you have increased both the time the service sleeps and the timeout. I think the sleep time should stay the same (105), the timeout  should be updated to 11 seconds. This way the difference is still one second as it used to be

Reply:
I am not sure to understand, I set the timeout to 5 seconds in`FtService`, so the difference between the sleep and timeout is actually 5 seconds, right ?
I thought after 5 seconds of sleep, timeout is triggered and response sent, so having client waiting for 11 seconds was reasonable.

Reply:
OK, I see it now. Thanks.